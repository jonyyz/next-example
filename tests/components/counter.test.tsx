import { render } from "@testing-library/react";
import StoreProvider from "@/store/index.js";
// @ts-ignore
import Counter from "@/components/counter.tsx";

describe("Counter Component", () => {
  it("should render", () => {
    // TODO: MockStoreProvider
    const { container } = render(
      <StoreProvider>
        <Counter message="test" />
      </StoreProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
