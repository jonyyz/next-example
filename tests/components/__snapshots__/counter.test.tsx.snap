// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Component should render 1`] = `
<div>
  <div>
    Counter: 
    43
  </div>
</div>
`;
